libcontainer/container_linux.go

```diff
@@ -523,10 +544,39 @@ func (c *linuxContainer) newInitProcess(p *Process, cmd *exec.Cmd, messageSockPa
 		}
 	}
 	_, sharePidns := nsMaps[configs.NEWPID]
-	data, err := c.bootstrapData(c.config.Namespaces.CloneFlags(), nsMaps)
+	data, err := c.bootstrapData(c.config.Namespaces.CloneFlags(), nsMaps, initStandard)
 	if err != nil {
 		return nil, err
 	}
+
+	if c.shouldSendMountSources() {
+		var mountFileFdsList strings.Builder
+
+		// We know the size of the string can easily len(c.config.Mounts). We multiply that by two, assuming
+		// the number we append has 2 digits top, to reduce allocations in most cases.
+		mountFileFdsList.Grow(len(c.config.Mounts) * 2)
```

Comment:
I think you forgot to take `;` into account.

Reply:
Yes, I completely forgot! And as you pointed out, that introduced a bug. Will be fixed when I push in a few minutes :)